ask_vopros.admin.inc

```diff
@@ -10,6 +10,11 @@
  */
 function ask_vopros_settings($form, &$form_state) {
   // If ting is configured with an agency id, just use that.
+  /**
+   * @TODO: I don't think we should force a agency id (special in tests it might
```

Comment:
Why would you want to test with another agency than the agency of your test site?

There's quite enough textfields in the Drupal backend as it is, and having another place to fill in the agency_id, the cognetive load on everybody that visits the config page, and the fact that I'll make the module *not* work out of the box in most cases, just doesn't seem worth the added convenience for the few.

I've just pushed some minor cleanup that as a side-effect ensures the ask_vopros_agency_id will take precedence, so it can be overridden using Drush.

In 99% of case you *do* want to use the agency_id if available.

If you insist, I'd rather keep the logic and show the used agency_id (which might be a good idea anyway), and add a "change" link to manually set the ask_vopros_agency_id (much like how the machine name widget works).